_includes/custom-head.html

```diff
@@ -0,0 +1,3 @@
+{% comment %}
+  A placeholder to allow defining custom metadata
+{% endcomment %}
```

Comment:
```suggestion
<!-- A placeholder to allow defining custom metadata -->
```
Comments using HTML style will be more compact.

Reply:
Accepted